content/en/tracing/trace_collection/trace_context_propagation/php.md

```diff
@@ -26,9 +29,9 @@ You can use the following environment variables to configure the PHP tracing lib
 - `DD_TRACE_PROPAGATION_STYLE_INJECT=Datadog,tracecontext,B3 single header`
 - `DD_TRACE_PROPAGATION_STYLE_EXTRACT=Datadog,tracecontext,B3 single header`
 
-The environment variable values are comma-separated lists of header styles enabled for injection or extraction. By default, only the `tracecontext` and `Datadog` injection styles are enabled.
+The environment variable values are comma-separated lists of header styles enabled for injection or extraction. The default style setting is `tracecontext,Datadog`.
 
-If multiple extraction styles are enabled, the extraction attempt is completed with the following priorities: `tracecontext` has priority, then `Datadog`, then B3.
+If multiple extraction styles are enabled, the extraction attempt is completed with the following priorities: `tracecontext` has priority, then `Datadog`, then `B3`.
```

Comment:
@bwoebi can you confirm whether the extraction is now order-dependent? If that's the case, then we should be able to remove this line from the documentation, right?

Reply:
Or we could take a line from the Ruby documentation that says: "If multiple extraction styles are enabled extraction attempt is done on the order those styles are configured and first successful extracted value is used."

Reply:
Or we could change it to the following line, taken from the Java page:

When multiple extraction styles are enabled, the extraction attempt is done on the order those styles are configured, using the first successful extracted value. If later valid trace contexts are found, they are terminated and appended as span links. Moreover, if the `tracecontext` style is enabled, W3C Tracestate is propagated if W3C Traceparent matches the extracted context.

Reply:
I got feedback from Bob offline, it's now done in the specified order, so we can change the text to the following:

```suggestion
If multiple extraction styles are enabled, the extraction attempt is done on the order those styles are configured and first successful extracted value is used.
```